tutorials/networking/websocket.rst

```diff
@@ -38,59 +38,32 @@ This example will show you how to create a WebSocket connection to a remote serv
 
     extends Node
 
-    # The URL we will connect to
-    export var websocket_url = "wss://libwebsockets.org"
-
     # Our WebSocketClient instance
-    var _client = WebSocketClient.new()
+    var _client = WebSocketPeer.new()
 
     func _ready():
-        # Connect base signals to get notified of connection open, close, and errors.
-        _client.connection_closed.connect(_closed)
-        _client.connection_error.connect(_closed)
-        _client.connection_established.connect(_connected)
-        # This signal is emitted when not using the Multiplayer API every time
-        # a full packet is received.
-        # Alternatively, you could check get_peer(1).get_available_packets() in a loop.
-        _client.data_received.connect(_on_data)
-
         # Initiate connection to the given URL.
-        var err = _client.connect_to_url(websocket_url, ["lws-mirror-protocol"])
+        # remote ip addr and port
```

Comment:
```suggestion
        # Remote address and port.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide). 

Also there's no IP address provided in this example so specifying IP would be confusing